src/browser/main.js

```diff
@@ -11,6 +11,9 @@ import {Provider} from 'react-redux';
 // TODO: Add app storage example.
 // import storage from 'redux-storage';
 
+// Enabling ES7 `async/await` in browser:
+if (process.env.IS_BROWSER) require('regenerator/runtime');
```

Comment:
I think we don't need `if (process.env.IS_BROWSER)` at this place since is called only from browser.